services/ui-src/src/utils/forms/schemas.ts

```diff
@@ -9,16 +9,55 @@ export const textOptional = () => text().notRequired();
 // NUMBER
 const validNAValues = ["N/A", "Data not available"];
 
-export const number = () =>
+const ignoreCharsForSchema = (value: string, charsToReplace: RegExp) => {
+  return numberSchema().transform((_value) => {
+    return Number(value.replace(charsToReplace, ""));
+  });
+};
+
+export const number = (isRatio = false) =>
```

Comment:
Is it possible to make a separate ratio field that validates against number : number ? I feel like that'd be pretty smooth

Reply:
Separated out!

Reply:
I'm glad this is separated, though my idea was wondering if it's possible to do like
`ratio: mixed(number(), ":", number())` if that makes sense. Like having the schema validate there are two numbers with a colon. Maybe there's no easy way to do that except for regex. Just curious since that would simplify it a lot!

Reply:
Not to my knowledge, no :( .. And if i try to do something like that then I'd come back to the complexity before I separated it out.